### ISSUE TITLE
date picker has duplicate labels when multiple on a page

### ISSUE THREAD

**Issue by helennickols** [ministryofjustice/moj-frontend#2210]
## What

The datepicker calendar icon has a hardcoded label of "Choose date".  If multiple datepickers are added to one page, this will result in duplicate labels, which is a WCAG violation.

## Why

Potential for WCAG violations in live services if multiple date pickers are used on one page.

## Who needs to be involved

- Content Designer
- Developer
- Accessibility Specialist

## Done

Fix has been done and communicated.
